guardian/support-dotcom-components
# Designable Banner

## What does this change?

This PR adds a new banner component `DesignableBanner` which supports a limited design being passed in as props. The design is configured in the RRCP. Currently only the images are in scope for the design, but this will be increased to cover all colours etc.

When a banner variant has a design (rather than a template) the design is fetched from Dynamo (actually these are prefetched asynchronously, following the existing pattern) and added to the props returned by SDC to be passed to the component.

## How to test

Configure a design in the RRCP:

<img width="1506" alt="Screenshot 2023-09-07 at 09 41 08" src="https://github.com/guardian/support-dotcom-components/assets/379839/a58e0c0f-ba7a-4866-aaa4-67c3ae39d404">

Select the design for a banner variant:

<img width="899" alt="Screenshot 2023-09-07 at 09 52 01" src="https://github.com/guardian/support-dotcom-components/assets/379839/64bc10fc-f665-4fe8-9dde-de72d8688c43">

See the new `DesignableBanner` component rendered, using the design:

<img width="1510" alt="Screenshot 2023-09-07 at 09 54 44" src="https://github.com/guardian/support-dotcom-components/assets/379839/0c4ee792-a77c-4494-8167-035077d2e46b">

## How can we measure success?



## Have we considered potential risks?



## Images



## Accessibility
